### PR TITLE
IGNITE-17585 fix flaky ItCommonApiTest.testSessionExpiration test

### PR DESCRIPTION
**Pull request by ygerzhedovich**

### CODE REVIEW

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java [hunk 1]
```diff
@@ -62,14 +62,14 @@ public void testSessionExpiration() throws Exception {
         sql("CREATE TABLE TST(id INTEGER PRIMARY KEY, val INTEGER)");
         sql("INSERT INTO TST VALUES (1,1), (2,2), (3,3), (4,4)");
 
-        Session ses1 = sql.sessionBuilder().defaultPageSize(1).idleTimeout(1, TimeUnit.MILLISECONDS).build();
+        Session ses1 = sql.sessionBuilder().defaultPageSize(1).idleTimeout(1, TimeUnit.SECONDS).build();
```
Comment (zstan) [inline modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java:65]: I found : public static final long SESSION_EXPIRE_CHECK_PERIOD = TimeUnit.SECONDS.toMillis(1);
Why do we need hardcoded: **long timeout** ? Also comment : "SessionManager.checkPeriod" confusing.

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java [hunk 2]
```diff
@@ -62,14 +62,14 @@ public void testSessionExpiration() throws Exception {
...
         Session ses2 = sql.sessionBuilder().defaultPageSize(1).idleTimeout(100, TimeUnit.SECONDS).build();
 
         ResultSet rs1 = ses1.execute(null, "SELECT id FROM TST");
         ResultSet rs2 = ses2.execute(null, "SELECT id FROM TST");
 
         // waiting for run session cleanup thread
-        Thread.sleep(timeout);
+        Thread.sleep(timeout + ses1.idleTimeout(TimeUnit.MILLISECONDS));
```
Comment (zstan) [inline modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java:72]: why do we need **ses1.idleTimeout(TimeUnit.MILLISECONDS)** addition here ?

File: modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/QueryProcessor.java [hunk 3]
```diff
@@ -47,6 +48,14 @@ public interface QueryProcessor extends IgniteComponent {
      */
     CompletableFuture<Void> closeSession(SessionId sessionId);
 
+    /**
+     * Provide list af live session.
```
Comment (zstan) [inline modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/QueryProcessor.java:52]: tupo

**Review by zstan — APPROVED** [review]
LGTM, fix typo and obtain green viza.

File: modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/QueryProcessor.java (hunk 3 above)
Comment (ygerzhedovich) [reply modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/QueryProcessor.java]: fixed

**Pull request closed without merging**